packages/framework/src/hooks/useStyles.ts

```diff
@@ -49,16 +50,20 @@ export const useStyles = <T extends { [key: string]: unknown }>(
     classStylesEval
       ?.trim()
       .split(" ")
+      .filter((x) => x)
       .map((className) => `.${className}`),
   )
     .join(" ")
     .concat(` ${namedStylesEval?.trim()}`);
 
   const styleProperties = useMemo(() => {
     return styleNames && themeContext
-      ? resolveStyleNames(styleNames.split(" "), themeContext)
+      ? resolveStyleNames(
+          [...styleNames.split(" "), widgetName || ""].filter((x) => x.trim()),
```

Comment:
Need to be careful about order here - stylenames override widget styles. Also choose a better variable other than x or use `compact` or `filter(isEmpty)`